.github/workflows/ci.yml

```diff
@@ -50,6 +50,9 @@ jobs:
               -DPYBIND11_FINDPYTHON=ON
           - runs-on: macos-latest
             python: pypy-2.7
+          # Inject a Windows 2019 run
+          - runs-on: windows-2019
+            python: 3.9
```

Comment:
Nice! I didn't know it's that easy.
While we're still advertising Python 2.7 support, could we add that in here, too?